### PR TITLE
Add birthday with nlp input

### PR DESCRIPTION
**Pull request by louislai**

### CODE REVIEW

File: src/seedu/addressbook/data/person/Birthday.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+/**
+ * Represents a Person's birthday in the address book.
+ * Guarantees: immutable;
+ */
+public class Birthday {
+    public final Date value;
+    private boolean isPrivate;
+
+    public static String EXAMPLE = "Sat Feb 18 16:03:28 SGT 2017";
```
Comment (Melvin-Tan) [inline src/seedu/addressbook/data/person/Birthday.java:18]: Is this too specific for birthday? Usually we only wanna know DD MM YYYY right?

File: src/seedu/addressbook/storage/jaxb/AdaptedPerson.java [hunk 2]
```diff
@@ -62,6 +64,10 @@ public AdaptedPerson(ReadOnlyPerson source) {
         address.isPrivate = source.getAddress().isPrivate();
         address.value = source.getAddress().value;
 
+        birthday = new AdaptedContactDetail();
+        birthday.isPrivate = source.getBirthday().isPrivate();
+        birthday.value = source.getBirthday().toString();
```
Comment (Melvin-Tan) [inline src/seedu/addressbook/storage/jaxb/AdaptedPerson.java:69]: Might want to make it consistent with the other columns like address/email/phone and just return a string value instead?

File: src/seedu/addressbook/data/person/Birthday.java (hunk 1 above)
Comment (louislai) [reply src/seedu/addressbook/data/person/Birthday.java:18]: lol this is so that the test case can return the exact same Date (including time)

Comment (louislai) [reply src/seedu/addressbook/data/person/Birthday.java:18]: maybe there s a `Day` type lying around somewhere?

File: test/java/seedu/addressbook/storage/StorageFileTest.java [hunk 3]
```diff
@@ -10,11 +10,7 @@
 
 import seedu.addressbook.data.AddressBook;
 import seedu.addressbook.data.exception.IllegalValueException;
-import seedu.addressbook.data.person.Address;
-import seedu.addressbook.data.person.Email;
-import seedu.addressbook.data.person.Name;
-import seedu.addressbook.data.person.Person;
-import seedu.addressbook.data.person.Phone;
+import seedu.addressbook.data.person.*;
```
Comment (Melvin-Tan) [inline test/java/seedu/addressbook/storage/StorageFileTest.java:13]: "3. Imported classes should always be listed explicitly."

**Pull request merged**